src/resources/postcss/customizer-controls.pcss

```diff
@@ -0,0 +1,304 @@
+/* = Customizer Controls (V2)
+============================ */
+
+.wp-customizer {
+
+	[id^="customize-control-tribe_customizer"] {
+
+		.customize-control-description {
+			font-style: normal;
+			margin-bottom: 12px;
+		}
+
+		.customize-control-title {
+			margin-bottom: 0;
+
+			& ~ .customize-control-content {
+				margin-top: 8px;
+			}
+
+			& ~ .customize-control-description {
+				margin-top: 4px;
+				margin-bottom: 3px;
+			}
+		}
+
+		.customize-control .customize-inside-control-row {
+			padding: 3px 0;
+		}
+
+		.button.wp-color-result {
+			border-color: #141827;
+
+			&:active,
+			&:focus,
+			&:hover {
+				border-color: #141827;
+				box-shadow: 0px 0px 0px .75px rgba(20, 24, 39, .5);
+			}
+		}
+
+		/* This handles a hidden color control based on a radio option - brings it closer to the radio control. */
+		/* Note it requires the radio name end in "_choice" and the color control end in "_color" */
+		&[id$="_choice"].customize-control-radio + [id^="customize-control-tribe_customizer"][id$="_color"] {
+			margin-top: -18px;
+		}
+
+
+		/* Range Slider, TEC-style */
+
+		.tec-range-slider-label {
+			border: 0;
+			clip: rect(0 0 0 0);
+			height: 1px;
+			margin: -1px;
+			overflow: hidden;
+			padding: 0;
+			position: absolute;
+			width: 1px;
+		}
+
+		input[type=range].tec-range-slider {
```

Comment:
a lot of these styles look like they are copied from the range slider base postcss, could we not import those in and add those classes to the range slider instead?

Reply:
Hm, maybe - I was trying to keep this as "light" as possible for now. Let me see what I can do.